.buildkite/pipeline.yml

```diff
@@ -132,20 +132,20 @@ steps:
         git submodule update --init --recursive
 
         # hard link pm2 logs to current dir, so buildkite can pick up them as artifacts
-        mkdir -p ~/.rainbow/logs/eth-relay
-        mkdir -p ~/.rainbow/logs/near-relay
+        mkdir -p ~/.rainbow/logs/eth2near-relay
```

Comment:
This part is no long need, i did some refactor e2e test to remove these ugly log collecting